### ISSUE TITLE
Wondering more details about finetuning RoBERTa on PhysicalIQA

### ISSUE THREAD

**Issue by taoshen58** [isi-nlp/ai2#1]
Hi @ChenghaoMou ,

Thanks for your implementation details on benchmarks from AI2. I'm trying to finetuning RoBERTa on PhysicalIQA and I want to know some more details:
1) what's the dev accuracy of the model that you submitted to leaderboard. 
2) what's the hyper-parameters when you were training the model. 

Thanks again! 

Best
Tao

**Comment by ChenghaoMou** [comment]
@taoshen58 

Hi, you can check the dev score on the right side of the page when you click each submission on the leaderboard. 

As for the hyper-parameters, they are stored in `config/hyparams.yaml` in this repo.

I am closing this issue for now. If you have any other questions, feel free to open another one.
